src/containers/PermissionsApplication.jsx

```diff
@@ -29,7 +29,21 @@ const PermissionsApplication = ({ match }) => {
           {t('Permissions.failedRequest')}
         </Typography>
       ) : (
-        <PageTitle>{appName}</PageTitle>
+        <div>
+          <PageTitle>{appName.toUpperCase()}</PageTitle>
+          {Object.entries(queryResult.data[0].attributes.permissions).map(
+            ([key, value]) => (
```

Comment:
i think the commit should be named like the name of the trello story. We are not only displaying permissions, but also permissions details, inside PermApplication page.